themes/scss/map/_overlays.scss

```diff
@@ -228,7 +228,11 @@ $buttonColor: #636D72;
 $maxLegendContainerHeight: 300px;
 
 .CDB-LayerLegends {
-  margin-bottom: 12px;
+  margin-top: 0;
+
+  + .CDB-LayerLegends {
```

Comment:
The same selector?

Reply:
Yes, so it's only applied when there is more than 1 legend